src/plugins/enum/README.md

```diff
@@ -0,0 +1,19 @@
+- infos = Information about the enum plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/placements = presetstorage
+- infos/description =
+
+## Introduction ##
+
+The Enum plugin checks string values of Keys by comparing it against a list of valid values.
+
+## Usage ##
+
+The plugin checks every Key in the Keyset for the Metakey `check/enum` containing a list with the syntax `['string1', 'string2', 'string3', ..., 'stringN']` and compares each value with the string value of the Key. If no match is found an error is returned.
```

Comment:
About the syntax: Why is it surrounded with []? It actually does make sense if you also can specify an enum somewhere else (e.g. in the plugin configuration) and reuse it `check/enum` (then without square brackets).

I am wondering how fast the check will perform with an enum of 700 values, copied to 700 places? A reference to the plugins configuration, and not copying the enum values, is certainly faster, but maybe its a premature optimization.